addons/sys_bf888s/radio/fnc_handleMultipleTransmissions.sqf

```diff
@@ -0,0 +1,198 @@
+#include "script_component.hpp"
+/*
+ * Author: ACRE2Team
```

Comment:
I would not use the  singleChannelPRR mode in here. That would prevent this radio to receive/send  transmissions to/from a SEM-52 / PRC77 etd

Reply:
I started switching over to using the method that the 77 etc uses and it seems to have broken the ability to communicate. I need to take a deeper look to make sure I didn't break it via something else.
Will I need to make modifications anywhere else? I don't seem to see any other references to singleChannelPRR in any of the the initialisation or preset stuff.  

Is this method even called for 1 person talking to 1 other person (which was the extent of my testing last night)

Reply:
You have to set the presets accordingly. At the moment you are using the PRC343 presets. The other radios use a different format

Reply:
Thanks, that helped me figure it out. I've switched it over to using `singleChannel`